pages/releases/[slug].js

```diff
@@ -0,0 +1,26 @@
+import { supabase } from "@/utils/supabase"
+import ReleaseLayout from "@/components/Releases/ReleaseLayout"
+export async function getServerSideProps({ params }) {
```

Comment:
This is so silly but... can you put an empty line between the imports and exported function?